templates/repo/branch/list.tmpl

```diff
@@ -22,7 +22,10 @@
 								{{if .DefaultBranchBranch.IsProtected}}
 									{{svg "octicon-shield-lock"}}
 								{{end}}
-								<a href="{{.RepoLink}}/src/branch/{{PathEscapeSegments .DefaultBranchBranch.DBBranch.Name}}">{{.DefaultBranchBranch.DBBranch.Name}}</a>
+								<div class="branch-detail">
```

Comment:
`class="flex-text-block"`

Reply:
Problem:- 
<img width="686" alt="image" src="https://github.com/go-gitea/gitea/assets/80308335/9504b463-2d29-41f7-a5f3-8157d374142c">

`flex-wrap` is creating problem

<img width="1134" alt="image" src="https://github.com/go-gitea/gitea/assets/80308335/6f36fcb9-0e65-45f7-a028-49704866b642">

Reply:
Then use `class="gt-df gt-ac"` instead ? These are single style helper CSS classes.

Reply:
From where I can get this cheat sheet.

Reply:
some are in `helper.css`

some are on the "devtest" page ( `http://localhost:3000/devtest`  in dev mode)